app/locale/ru.coffee

```diff
@@ -1196,14 +1196,14 @@ module.exports = nativeDescription: "русский", englishDescription: "Russi
     enrollment_credits_available: "Доступно лицензий:"
     language_select: "Выберите язык" # ClassroomSettingsModal
     language_cannot_change: "Язык не может быть изменён после того, как ученик присоединился к классу."
-#    avg_student_exp_label: "Average Student Programming Experience"
-#    avg_student_exp_desc: "This will help us understand how to pace courses better."
-#    avg_student_exp_select: "Select the best option"
-#    avg_student_exp_none: "No Experience - little to no experience"
-#    avg_student_exp_beginner: "Beginner - some exposure or block-based"
-#    avg_student_exp_intermediate: "Intermediate - some experience with typed code"
```

Comment:
Silly question: what does "typed code" means here?:)
"code written by pressing the keys." or "code with strong or weak typing"?